src/dmd/json.d

```diff
@@ -47,13 +48,20 @@ private extern (C++) final class ToJsonVisitor : Visitor
 public:
     OutBuffer* buf;
     int indentLevel;
-    const(char)* filename;
+    const(char)[] filename;
 
     extern (D) this(OutBuffer* buf)
     {
         this.buf = buf;
     }
 
+    /// Helper: Remove when all usages of strlen are gone
+    static extern(D) const(char)[] toDString (const(char)* s)
+    {
+        return s ? s[0 .. strlen(s)] : null;
+    }
```

Comment:
Why not put this as a global function into `utils` or similar? It's used at other places too and doesn't really fit in `ToJsonVisitor`

Reply:
Didn't expect it to be so widely used :(
Also I'm wary that `dmd.utils` will be imported everywhere soon.
But let's give it a shot...